server/lib/pbench/process_tb.py

```diff
@@ -95,9 +100,13 @@ def process_tb(self) -> Results:
             tb = Path(str(tbmd5)[0 : -len(".md5")])
             tbdir = tb.parent
             controller = tbdir.name
+            satellite_prefix = ""
```

Comment:
This works, but it makes the optional nature of the `satellite_prefix` parameter useless in this case.  Consider defaulting to `None`.

Reply:
Defaulting to `None` won't exactly trigger the default value of the prefix inside `_results_push()`, but it will do what I think we want (which this code doesn't, quite).  (Good catch.)

Reply:
Did we decide that default to `None` was not good to do for some reason?

Reply:
I think we concurred with you:  `satellite_prefix` should be initialized to `None` at line 103.

The only way to do "better" would be to code two separate calls to `_results_push()`, one with the `satellite_prefix` and one without, but I don't think that the value that produces is worth the complication.

Reply:
(My apologies for being less than clear in my reply yesterday.)